App/person.py

```diff
@@ -1,28 +1,399 @@
+                    job_type = 'fellow'
+                    # check whether person has an office
+                    if username in list(Person.fellows_not_allocated_office):
+                        allocated_office = 'unallocated'
+                    else:
+                        for room in Room.total_rooms:
+                            if room in list(Room.offices):
+                                for occupant in Room.total_rooms[room]:
+                                    if person_id == occupant:
+                                        allocated_office = room
+
+                    # check whether fellow has a living space
+                    if username in list(Person.unallocated_people):
+                        is_accomodated = False
+                        allocated_livingspace = 'unallocated'
+                    else:
+                        is_accomodated = True
+                        for room in Room.total_rooms:
+                            if room in list(Room.livingspaces):
+                                for occupant in Room.total_rooms[room]:
+                                    if person_id == occupant:
+                                        allocated_livingspace = room
+
+                # if the person is a staff...
+                else:
+                    job_type = 'staff'
+                    allocated_livingspace = 'unallocated'
+                    if username in list(Person.staff_not_allocated_office):
+                        is_accomodated = False
+                        allocated_office = 'unallocated'
+                    else:
+                        for room in Room.total_rooms:
+                            if room in list(Room.offices):
+                                for occupant in Room.total_rooms[room]:
+                                    if person_id == occupant:
+                                        allocated_office = room
+
+                # commit person data as an object of AmityPerson class (database table for Person)
+                person_info = AmityPerson(person_id=person_id, username=username, job_type=job_type, is_accomodated=is_accomodated, allocated_livingspace=allocated_livingspace, allocated_office=allocated_office)
+
+                session.add(person_info)
+                session.commit()
+                session.close()
+
+            return "Person data commit successfull"
+
+        @staticmethod
+        def load_people(db_name):
+            """
+            method called to load person data from database tables to respective data structures
+            """
+            global engine
+            engine = create_engine('sqlite:///'+db_name, echo = False)
+            Session = sessionmaker()
+            Session.configure(bind=engine)
+            session = Session()
+            all_people = session.query(AmityPerson).all()
+            all_fellows = session.query(AmityPerson).filter_by(job_type="fellow")
```

Comment:
how about
`all_fellows = [person for person in all_people if person.job_type=="fellow" ]`